index.html

```diff
@@ -1 +1,12 @@
-<h1>Welcome to Microverse</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=>, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1></h1>
```

Comment:
- [ ] You should fill h1 tag according to [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.html%20file%20that%20includes%20a%20header%20with%20the%20text%20%22Hello%20Microverse!%22.%20Use%20h1%20tag%20for%20that.)